server/lib/settings.ts

```diff
@@ -356,6 +359,7 @@ class Settings {
       applicationTitle: this.data.main.applicationTitle,
       hideAvailable: this.data.main.hideAvailable,
       localLogin: this.data.main.localLogin,
+      newPlexLogin: this.data.main.newPlexLogin,
```

Comment:
Why is this being added to public settings?

Reply:
Oh yup, good call, it doesn't need to be.